src/utils/createAlertsContent.js

```diff
@@ -236,7 +244,7 @@ export const createAlertRowData = ({ ...alert }, isCrossProjects) => {
         id: `alertName.${alert.id}`,
         headerId: 'alertName',
         headerLabel: 'Alert Name',
-        value: name,
+        value: getName(alert.name, alert.event_kind).value,
```

Comment:
based on alert.name add a class name not inline styles

Reply:
Done